addon/components/lt-body.js

```diff
@@ -266,17 +267,31 @@ export default Component.extend({
       let isSelected = row.get('selected');
       let currIndex = rows.indexOf(row);
       let prevIndex = this._prevSelectedIndex === -1 ? currIndex : this._prevSelectedIndex;
+      let affectedSelections = Ember.A();
```

Comment:
I was originally going to use a native array here, but w/o prototype extensions turned on, I'm unable to use ember's convenience methods such as `pushObjects()`.  Also since this array is being sent to component consumer via the action, I wanted to make sure they are always getting a nice Ember array regardless of their environment configuration.